stratum/hal/lib/tdi/tofino/tofino_chassis_manager.cc

```diff
@@ -94,82 +96,95 @@ ::util::Status TofinoChassisManager::AddPortHelper(
 
   const auto& config_params = singleton_port.config_params();
   if (config_params.admin_state() == ADMIN_STATE_UNKNOWN) {
-    RETURN_ERROR(ERR_INVALID_PARAM)
-        << "Invalid admin state for port " << port_id << " in node " << node_id
-        << " (SDK Port " << sdk_port_id << ").";
-  }
+    return MAKE_ERROR(ERR_INVALID_PARAM)
+        << "Invalid admin state for port " << port_id << " in node "
+		<< node_id << " (SDK Port " << sdk_port_id << ").";
```

Comment:
Broken indentation (maybe a tab in the file?)

Reply:
fixed